Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Block-List-Editor/index.md

```diff
@@ -12,6 +12,8 @@ versionFrom: 8.7.0
 
 :::note
 A default Umbraco CMS installation does not ship with a defined *Data Type* using the Block List editor. In order to start using the property, follow the steps outlined below.
+
+<a href="https://umbraco.com/blog/deep-dive-the-block-list-editor/">Click here for an overview with a worked example and references back to the relevant documention.</a>
```

Comment:
Small suggestion here - did you mean "working" instead of "worked"?

```suggestion
<a href="https://umbraco.com/blog/deep-dive-the-block-list-editor/">Click here for an overview with a working example and references back to the relevant documentation.</a>
```

Reply:
Hi @sofietoft, no I meant "worked". It implies you can see the working out of the solution as it's developed across the article. Changing it to "working" would imply it's a solution you can download and see it working before your eyes, which is not what I meant. Not sure how to remove your suggestion as it only gives the option to commit, which is a bit presumptuous on GitHub's part, lol